filament/backend/src/opengl/OpenGLContext.cpp

```diff
@@ -178,6 +181,9 @@ OpenGLContext::OpenGLContext() noexcept {
             // early exit condition is flattened in EASU code
             // (that should be regardless of ANGLE, but we should double check)
             bugs.split_easu = true;
+
+            // double check like split_easu above
+            bugs.enable_initialize_non_used_uniform_array = true;
```

Comment:
I think this should be `false` here, because this bug doesn't exist when using the Vulkan backend (Angle is using Vulkan).